config.go

```diff
@@ -217,10 +217,7 @@ func generateNewClient(c *Config, client *alks.Client) (*alks.Client, error) {
 		return nil, err
 	}
 
-	newClient, err := alks.NewSTSClient(c.URL, newCreds.AccessKey, newCreds.SecretKey, newCreds.SessionToken)
-	if err != nil {
-		return nil, err
-	}
```

Comment:
Can you either explain why we removed this error handling or add it back in?

Reply:
Inside of NewSTSClient, there is no error being returned it just gets ignored, so removing this block will clean up unused code and removes the need for us to change this to use AlksError since there is no error returned. We can change it to use AlksError but i thought it was unnecessary.